### DISCUSSION TITLE
[Idea] Make Feature Row with Two Card centered

### DISCUSSION THREAD

**Discussion by ghost** [mmistakes/minimal-mistakes#3079] [Ideas]
## Environment

- Minimal Mistakes version: 4.24.0
- Ruby gem or remote theme version: `ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x64-mingw32]`
- Jekyll version:  `jekyll 4.2.0`
- Git repository URL: [jekyll.zip](https://github.com/mmistakes/minimal-mistakes/files/6872651/jekyll.zip)
- GitHub Pages hosted (if yes provide URL to site): Self Hosted
- Operating system: Windows 10 21H1 19043.1110

## Expected behavior
As I said on #3076, I am selfhosting my website with Jekyll. I am using Minimal Mistakes with the Feature Row on my Home Page. I am using  `type="left"` on `feature_row` mainly because of this issue.
Basically, if you're using two cards on the Feature Row instaed of three, they will not look like centered. I thought they would.

## Steps to reproduce the behavior
* Add two card to the Feature Row
* Insert `{% include feature_row %}` at the end of the page
* Refresh

## Other
![image](https://user-images.githubusercontent.com/52204908/126873395-e4f00709-3b4b-49d1-9709-928d5ac53fd7.png)

**Comment by mmistakes** [comment]
Feature rows currently only support 1 block or 3 in a row.
An enhancement would be needed to rewrite the outdated `float` based CSS and use something more flexible and modern like `display: flex` or grid.

https://mmistakes.github.io/minimal-mistakes/splash-page/
